### ISSUE TITLE
imagenet result of CAE-large model with 800 epochs

### ISSUE THREAD

**Issue by edhuang1** [lxtGH/CAE#17]
Dear authors,

Thank you for your excellent work.
Could you also provide the accuracy of ViT-Large in an 800-epoch pre-training scheme?

Thank you for your grateful help

Best wishes,

**Comment by charlesCXK** [comment]
Linear probing: 76.3
Atentive probing: 80.0
ImageNet finetuning: 86.0
Semantic Segmentation: 54.4
